user-service/test/java/controllers/BaseControllerTest.java

```diff
@@ -0,0 +1,31 @@
+package controllers;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BaseControllerTest {
```

Comment:
In play for controller test we have implement play helper class for mocking and running play server.